frontend/src/pages/chat/Chat.tsx

```diff
@@ -565,6 +565,18 @@ const Chat = () => {
         return isLoading || (messages && messages.length === 0) || clearingChat || appStateContext?.state.chatHistoryLoadingState === ChatHistoryLoadingState.Loading
     }
 
+    const sendChatQuestion =  (question: string, id?: string | undefined) => {
+      appStateContext?.state.isCosmosDBAvailable?.cosmosDB ? makeApiRequestWithCosmosDB(question, id) : makeApiRequestWithoutCosmosDB(question, id)
+    }
+    
+    useEffect(() => {
+      const urlParams = new URLSearchParams(window.location.search);
```

Comment:
Is `URLSearchParams` native to TS/JS? I don't see any imports or class definitions.

Reply:
URLSearchParams is a native JS API and does not require an import.